wizeline/project-lab
# Feat/looking for field-21- Implement the “Looking for” list in ProjectDetail page.

#### What does this PR do?
Implement the "Looking for" field, to add the required or necessary disciplines in the project, this PR adds the schema, seeds.

#### Where should the reviewer start?

app/pages/projects/[projectId]/index.tsx
app/disciplines/queries/getDisciplines.ts
app/projects/components/ProjectForm.tsx
app/projects/mutations/updateProject.ts
app/projects/mutations/createProject.ts
app/projects/mutations/deleteProject.ts

#### How should this be manually tested?

1. In the home page create a new Proposal
2. Fill the default fields and add more details
3. Fill the new field of "Looking For" to add the disciplines that the project need
4. Create the project, and you could see the disciplines under the skills area.


#### Screenshots

<img width="1365" alt="image" src="https://user-images.githubusercontent.com/97306904/163943306-7d4696ae-d31b-40e5-b862-561d28259d9f.png">

![image](https://user-images.githubusercontent.com/97306904/164078329-cc719a8f-02a0-4bd3-99e2-b5e88ded3406.png)

#### Checklist



- [x] I added the disciplines schema.
- [x] I added some discipline seeds.
- [x] I create the component to add, edit or delete disciplines in a project.
- [x] I update the mutations for create, update, delete a project. 

Issue #21
